src/test/suite/participant/participant.test.ts

```diff
@@ -0,0 +1,312 @@
+import * as vscode from 'vscode';
+import { beforeEach, afterEach } from 'mocha';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinon from 'sinon';
+import type { DataService } from 'mongodb-data-service';
+
+import ParticipantController, {
+  parseForDatabaseAndCollectionName,
+  getRunnableContentFromString,
+  QUERY_GENERATION_STATE,
+} from '../../../participant/participant';
+import ConnectionController from '../../../connectionController';
+import { StorageController } from '../../../storage';
+import { StatusView } from '../../../views';
+import { ExtensionContextStub } from '../stubs';
+import TelemetryService from '../../../telemetry/telemetryService';
+import { TEST_DATABASE_URI } from '../dbTestHelper';
+import { CHAT_PARTICIPANT_ID } from '../../../participant/constants';
+
+const expect = chai.expect;
+
+chai.use(chaiAsPromised);
+
+suite('Participant Controller Test Suite', function () {
+  const sandbox = sinon.createSandbox();
```

Comment:
I don't remember if we came to a conclusion in another thread 🙈 So do we want to avoid sandboxes? I am almost certain we used sinon directly and then rewrote everything with sandbox after someone's feedback. I don't mind either way!

Reply:
Totally don't remember that conversation 😂 
I think I prefer avoiding sandboxes, but I don't think it matters either way really. Compass has the same issue with about 50% of the time we're using sandbox and 50% not 🫠

Reply:
Removed this as part of this merged pr 👍